src/pkg/shell/shell.go

```diff
@@ -39,6 +40,20 @@ func Run(name string, stdin io.Reader, stdout, stderr io.Writer, arg ...string)
 	return nil
 }
 
+// RunGetOutput spawns a command, capturing its stdout.
+func RunGetOutput(name string, arg ...string) ([]byte, error) {
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+	exitCode, err := RunWithExitCode(name, nil, &stdout, &stderr, arg...)
+	if err != nil {
+		return nil, err
+	}
+	if exitCode != 0 {
+		return nil, fmt.Errorf("failed to invoke %s: %s", name, string(stderr.Bytes()))
```

Comment:
What about `err`? Does it hold exactly the same information like `stderr`?

Reply:
Mmm... nevermind, it is being handled in the previous if block.

Another concern could be multi-line output in `stderr`. Maybe print it with `logrus`?

Reply:
> What about err? Does it hold exactly the same information like stderr?

No, `err` is not set here.  The command executed, but returned an error exit code.

> Another concern could be multi-line output in stderr. Maybe print it with logrus?

Ultimately, this API is for cases where we're just acting as a thin wrapper around the underlying CLI.  In the end, all we do with this API is dump the resulting text to stderr anyways, so multi-line output is fine.